core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config.properties

```diff
@@ -23,7 +23,7 @@
 warning.core = Jenkins core: {0}
 warning.plugin = {0}: {1} 
 
-blurb = This section allows you to suppress currently applicable warnings reported \
-  by the update site from appearing in this Jenkins instance\u2019s update notifications
-  indicator. \
+blurb = This section allows you to suppress warnings, applicable to your Jenkins \
+  configuration, provided by the updates sites. If you do, they won't be shown by \
```

Comment:
@daniel-beck are there multiple _updates sites_ or is it just one _updates site_?

Reply:
There can be 1..many, although I expect the vast majority of instances to just have one.

Typically https://plugins.jenkins.io/update-sites-manager or specialty plugins such as https://plugins.jenkins.io/cloudbees-devoptics-enabler add additional update sites.

Reply:
thanks.